docs/architecture/adr-027-deterministic-protobuf-serialization.md

```diff
@@ -11,7 +12,7 @@ Proposed
 ## Context
 
 [Protobuf](https://developers.google.com/protocol-buffers/docs/proto3)
-seralization is not unique (i.e. there exist a practically unlimited number of
+serialization is not unique (i.e. there exist a practically unlimited number of
```

Comment:
So the major use-case is to sign a structure, correct? Flow:

1. Serialize structure -> raw bytes
2. hash raw bytes
3. sign the hash.

Let's add a description of the major use-case. This will make it easier to frame the context.

Reply:
That's outside the scope of this PR, which only aims to clarify the rules for serialization.

Reply:
Ok, but do you know the answer? What is the major use case for this ADR?

Reply:
I think the answer is exactly what you said above! I.e. canonical serialization for anything that's hashed or signed over.